cnxpublishing/views/admin.py

```diff
@@ -271,3 +534,150 @@ def admin_edit_site_message_POST(request):
     args = admin_edit_site_message(request)
     args['response'] = "Message successfully Updated"
     return args
+
```

Comment:
😕  Am I crazy or is the following the same set of functions as above just copied to the lower part of the file?